common/src/main/java/com/microsoft/identity/common/components/AndroidPlatformComponentsFactory.java

```diff
@@ -119,7 +119,6 @@ private static void fillBuilder(@NonNull final PlatformComponents.PlatformCompon
                                     @NonNull final Context context,
                                     @Nullable final Activity activity,
                                     @Nullable final Fragment fragment) {
-        builder.storageEncryptionManager(new AndroidAuthSdkStorageEncryptionManager(context));
         fillBuilderWithBasicImplementations(builder, context, activity, fragment);
```

Comment:
Is it possible to replace the fillBuilder call in line 110 with fillBuilderWithBasicImplementation, and then delete fillBuilder?